User-Stories.txt

```diff
@@ -0,0 +1,35 @@
+Functional representation of User Stories
```

Comment:
Great addition with this file. I like the way it illustrates your representation of the step by step user stories